src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -294,6 +303,12 @@
   <data name="MaybeInstallRootCertMessage" xml:space="preserve">
     <value>Adding certificates to the operating system's trusted root certificates is insecure. Ensure that the target store is not root store.</value>
   </data>
+  <data name="MaybeUseWeakKDFInsufficientIterationCount" xml:space="preserve">
+    <value>Ensure Sufficient Iteration Count When Using Weak Key Derivation Function</value>
+  </data>
+  <data name="MaybeUseWeakKDFInsufficientIterationCountMessage" xml:space="preserve">
+    <value>Default iteration count 1000 needs to be overwritten in this object creation. Ensure that it is using at least 100k iterations when deriving cryptographic key</value>
```

Comment:
>Default iteration count 1000 needs to be overwritten in this object creation. Ensure that it is using at least 100k iterations when deriving cryptographic key [](start = 11, length = 158)

Ensure that the iteration count is at least {0} when deriving a cryptographic key from a password. By default, Rfc2898DeriveByte's IterationCount is only {1}.

Reply:
The default IterationCount can be fixed as 1000? Cause it won't change.

Reply:
Oh, I was thinking of localizing 1000 as "1,000" or "1.000", but given that we're dealing with code here, just "1000" is fine.  So nevermind.